erigon-lib/downloader/torrent_files.go

```diff
@@ -53,9 +53,37 @@ func (tf *TorrentFiles) delete(name string) error {
 func (tf *TorrentFiles) Create(name string, res []byte) error {
 	tf.lock.Lock()
 	defer tf.lock.Unlock()
-	return tf.create(filepath.Join(tf.dir, name), res)
+	return tf.create(name, res)
 }
-func (tf *TorrentFiles) create(torrentFilePath string, res []byte) error {
+
+func (tf *TorrentFiles) CreateIfNotProhibited(name string, res []byte) (ts *torrent.TorrentSpec, prohibited, created bool, err error) {
```

Comment:
`prohibited` and `created` results are not used anywhere - are they going to be used in a future PR? if not then should we remove them from the func signature?

also looks like `created` is always returned as false - seems like we are missing a statement to set created to `true` and return that in the case of a file getting created